mod/aalborg_theme/views/default/elements/forms.css.php

```diff
@@ -250,6 +250,16 @@
 	background-color: #EEE;
 	display: block;
 }
+.ui-helper-hidden-accessible {
+	border: 0;
+	clip: rect(0 0 0 0);
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	width: 1px;
```

Comment:
jQuery UI theme are filled with junk, so I wouldn't be surprised that all of these are necessary to hide an element...